queries.sql

```diff
@@ -0,0 +1,68 @@
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species is null;
+
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT DEL_YEAR_2022;
+
+UPDATE animals
+SET weight_kg = (weight_kg * -1);
+
+ROLLBACK TO DEL_YEAR_2022;
+
+UPDATE animals
+SET weight_kg = (weight_kg * -1)
+WHERE weight_kg < 0;
+
```

Comment:
- [ ] It would be better if after the `Update all animals' weights that are negative to be their weight multiplied by -1` , we have to commit the transaction as seen in the project requirements
![Screenshot from 2023-06-13 19-15-34](https://github.com/haftamudesta/vet_clinic_database/assets/46388113/193afa71-fa6e-4bc0-8495-433f2aa6061e)